src/notation/view/widgets/editstyle.ui

```diff
@@ -12053,7 +12060,7 @@
                     </sizepolicy>
                    </property>
                    <property name="text">
-                    <string>Position:</string>
+                    <string>Placement:</string>
```

Comment:
About this one: we actually decided that we want to change all occurrences of Placement in the style dialog to Position. Those offset spin boxes should be called "Offset above" and "Offset below". So:
- Position: dropdown (options above/below)
- Offset above: spinbox
- Offset below: spinbox.

Reply:
Does that include renaming the corresponding widgets? Like `lyricsPlacement` to `lyricsPosition`?

Reply:
The plai text changes are easy and done